Resources/Prototypes/SimpleStation14/Entities/Structures/Storage/Crates/crates.yml

```diff
@@ -0,0 +1,11 @@
+- type: entity
+  parent: CrateBaseSecure
+  id: CrateFreezerSecure
+  name: secure freezer
+  components:
+    - type: Icon
+      sprite: SimpleStation14/Structures/Storage/Crates/freezer_secure.rsi
+    - type: Sprite
+      sprite: SimpleStation14/Structures/Storage/Crates/freezer_secure.rsi
+    - type: AccessReader
+      access: [["Kitchen"]]
```

Comment:
I am unsure of how wise it is to give Chefs alone access to several poisonous meats. Chef already has access to a _lot_ of murder tools.

Reply:
well uh what access should it go to then?

Reply:
Security was my thought initially. I'm not entirely against this, just uncertain.

Reply:
oh I didnt check the price